src/build/build.ts

```diff
@@ -45,8 +46,13 @@ process.on('uncaughtException', (err) => {
   console.error(err.stack);
 });
 
+process.on('unhandledRejection', (err, p) => {
```

Comment:
I like longer names, like `error` and `promise`. We only have to type them once, then read them foreverrrr...